### PR TITLE
update: added log to notation key and certificate commands

### PR DESCRIPTION
**Pull request by patrickzheng200**
This PR adds log to notation key and certificate commands and updated the corresponding spec in key.md and certificate.md.

This PR is a part of resolving https://github.com/notaryproject/roadmap/issues/71.

### CODE REVIEW

File: cmd/notation/key.go [hunk 1]
```diff
@@ -167,6 +179,7 @@ func addKey(command *cobra.Command, opts *keyAddOpts) error {
 	}
 	pluginName := opts.plugin
 	if pluginName != "" {
+		logger.Debug("Adding external key...")
 		key, err = addExternalKey(command.Context(), opts, pluginName, name)
```
Comment (JeyJeyGao) [inline cmd/notation/key.go:183]:
```suggestion
		key, err = addExternalKey(ctx, opts, pluginName, name)
```

Comment (patrickzheng200) [reply cmd/notation/key.go]: Nice catch!

**Review by JeyJeyGao — APPROVED** [review]
LGTM

**Review by JeyJeyGao — APPROVED** [review]

**Review by shizhMSFT — APPROVED** [review]
LGTM

File: cmd/notation/key.go [hunk 2]
```diff
@@ -167,7 +179,8 @@ func addKey(command *cobra.Command, opts *keyAddOpts) error {
 	}
 	pluginName := opts.plugin
 	if pluginName != "" {
-		key, err = addExternalKey(command.Context(), opts, pluginName, name)
+		logger.Debug("Adding external key...")
```
Comment (priteshbandi) [inline cmd/notation/key.go:182]: usually add more information/context before executing the command
```suggestion
		logger.Debug("Adding key with name %v and plugin name %v, key, pluginName")
```

File: cmd/notation/key.go [hunk 3]
```diff
@@ -186,6 +199,7 @@ func addKey(command *cobra.Command, opts *keyAddOpts) error {
 	}
 
 	// write out
+	logger.Debugf("Added external key: {Name: %s, ExternalKey: %+v}", key.Name, key.ExternalKey)
```
Comment (priteshbandi) [inline cmd/notation/key.go:202]:
```suggestion
	logger.Debugf("Added key with name %s - %v", key.Name, key.ExternalKey)
```

File: cmd/notation/cert/show.go [hunk 4]
```diff
@@ -33,7 +37,7 @@ func certShowCommand(opts *certShowOpts) *cobra.Command {
 			opts.cert = args[0]
 			return nil
 		},
-		Long: `Show details of a certain certificate file
+		Long: `Show certificate details given trust store type, named store, and certificate file name. If the certificate file contains multiple certificates, then all certificates are displayed
```
Comment (priteshbandi) [inline cmd/notation/cert/show.go:40]:
```suggestion
		Long: `Show certificate details of given trust store name, trust store type, and certificate file name. If the certificate file contains multiple certificates, then all certificates are displayed
```

File: cmd/notation/cert/show.go [hunk 5]
```diff
@@ -71,6 +80,7 @@ func showCerts(opts *certShowOpts) error {
 	if err != nil {
 		return fmt.Errorf("failed to show details of certificate %s, with error: %s", cert, err.Error())
 	}
+	logger.Debugln("Trying to show details of certificate:", path)
```
Comment (priteshbandi) [inline cmd/notation/cert/show.go:83]:
```suggestion
	logger.Debugln("Showing details of certificate:", path)
```

**Review by priteshbandi — APPROVED** [review]
LGTM with suggestions

**Pull request merged**